src/main/kotlin/io/github/yuutoproject/yuutobot/commands/base/AbstractCommand.kt

```diff
@@ -28,15 +28,17 @@ import org.slf4j.LoggerFactory
  * - [name]: The name of the command
  * - [category]: The category of the command
  * - [description]: Oneliner that shows in "y!help list"
- * - [usage]: Extended help message for the command with information on how to use it
+ * - [parameters]: Short string of all the parameters the command accepts
+ * - [notes]: Additional notes that the user may have to note
  *
  * Commands can add aliases by overriding the [aliases] prop
  */
 abstract class AbstractCommand(
     val name: String,
     val category: CommandCategory,
     val description: String,
-    val usage: String
+    val parameters: String,
+    val notes: String
```

Comment:
I would make these nullable and null by default because they are optional
```suggestion
    val parameters: ?String = null,
    val notes: ?String = null
```